AppCenterDataStorage/AppCenterDataStorageTests/MSTokenExchangeTests.m

```diff
@@ -136,7 +192,6 @@ - (void)testValidCachedTokenExists {
 
   // Then
   [self waitForExpectationsWithTimeout:5 handler:nil];
-  OCMVerifyAll(self.ingestionStoreMock);
```

Comment:
Why do you delete this one? This is verifying `OCMReject` here, you need to call this one for `httpMock`.

Reply:
Please look at http://ocmock.org/reference/ and @clpolet  confirmed `OCMReject` does not need that.
You are probably confused with `OCMExpect`.

Reply:
```
Re-throwing fail fast exceptions in verify all
In fail-fast mode an exception might not cause the test to fail. This can happen when the call stack for the method does not end in the test. Fail fast exceptions will be re-thrown when OCMVerifyAll is called. This makes it possible to ensure that unwanted invocations from notifications etc. can be detected.
```

Reply:
I tested without and it rejected the call.